docs/codeql/reusables/supported-frameworks.rst

```diff
@@ -306,6 +306,39 @@ and the CodeQL library pack ``codeql/ruby-all`` (`changelog <https://github.com/
    rubyzip, Compression library
    typhoeus, HTTP client
 
+Rust built-in support
+====================================
+
+Provided by the current versions of the
+CodeQL query pack ``codeql/rust-queries`` (`changelog <https://github.com/github/codeql/tree/codeql-cli/latest/rust/ql/src/CHANGELOG.md>`__, `source <https://github.com/github/codeql/tree/codeql-cli/latest/rust/ql/src>`__)
+and the CodeQL library pack ``codeql/rust-all`` (`changelog <https://github.com/github/codeql/tree/codeql-cli/latest/rust/ql/lib/CHANGELOG.md>`__, `source <https://github.com/github/codeql/tree/codeql-cli/latest/rust/ql/lib>`__).
+All support is experimental.
+
+.. csv-table::
+   :header-rows: 1
+   :class: fullWidthTable
+   :widths: auto
+   :align: left
+
+   Name, Category
+   alloc, Standard library
+   core, Standard library
```

Comment:
```suggestion
   core, Standard library
   std, Standard library
```
As of https://github.com/github/codeql/pull/19446 we will have models for `env`, `fs`, `io`, `net`, `option` and `result` from the [`std`](https://doc.rust-lang.org/std/index.html) part of the standard library, which I think is plenty to justify listing it here.